packages/node/src/sdk.ts

```diff
@@ -132,6 +132,11 @@ export function init(options: NodeOptions = {}): void {
   if (options.autoSessionTracking) {
     startSessionTracking();
   }
+
+  // TODO: Right now, unless the user sets this option, this is a no-op, since 10 is Node's default value. Written this
+  // way so that it's easy to bump it if and when we choose to do so (would be a breaking change since it would likely
+  // have an effect on grouping)
+  Error.stackTraceLimit = typeof options.maxStacktraceFrames === 'number' ? options.maxStacktraceFrames : 10;
```

Comment:
What do you think about extracting `10` as a variable?

Errors may be very uninformative if the stack is full, should we also set the stack size (`--stack-size`)? See [second paragraph](https://github.com/nodejs/node/issues/32221#issuecomment-598068685). On the other hand, should we check about users setting `0` or negative values?

Reply:
See [this comment](https://stackoverflow.com/questions/11332422/how-can-i-increase-the-maximum-call-stack-size-in-node-js#comment89241038_11332459) and others around it. I think we shouldn't mess with `--stack-size`, because what values are safe is going to depend on the OS and picking the wrong value can crash the whole node process. (Besides, the default values mentioned there are so much higher than the number of frames we're showing that it's unlikely we'd ever run into that limit anyway.)